financial-connections/src/main/java/com/stripe/android/financialconnections/navigation/DestinationMappers.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.financialconnections.navigation
+
+import androidx.navigation.NavDestination
+import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest.Pane
+
+private val paneToDestination = mapOf(
+    Pane.INSTITUTION_PICKER to Destination.InstitutionPicker,
+    Pane.CONSENT to Destination.Consent,
+    Pane.PARTNER_AUTH to Destination.PartnerAuth,
+    Pane.ACCOUNT_PICKER to Destination.AccountPicker,
+    Pane.SUCCESS to Destination.Success,
+    Pane.MANUAL_ENTRY to Destination.ManualEntry,
+    Pane.ATTACH_LINKED_PAYMENT_ACCOUNT to Destination.AttachLinkedPaymentAccount,
+    Pane.NETWORKING_LINK_SIGNUP_PANE to Destination.NetworkingLinkSignup,
+    Pane.NETWORKING_LINK_LOGIN_WARMUP to Destination.NetworkingLinkLoginWarmup,
+    Pane.NETWORKING_LINK_VERIFICATION to Destination.NetworkingLinkVerification,
+    Pane.NETWORKING_SAVE_TO_LINK_VERIFICATION to Destination.NetworkingSaveToLinkVerification,
+    Pane.LINK_ACCOUNT_PICKER to Destination.LinkAccountPicker,
+    Pane.LINK_STEP_UP_VERIFICATION to Destination.LinkStepUpVerification,
+    Pane.RESET to Destination.Reset,
+    Pane.MANUAL_ENTRY_SUCCESS to Destination.ManualEntrySuccess,
+)
+
+internal val Pane.destination: Destination
```

Comment:
- Went with the `Map` since it allows mapping bidirectionally (we're also mapping destination to pane [here](https://github.com/stripe/stripe-android/blob/554f48bb5e42da54c284cc50e3b3d263377bda68/financial-connections/src/main/java/com/stripe/android/financialconnections/navigation/DestinationMappers.kt#L28))
- Also added a test that ensures all panes are mapped, except for the ones with no matching UI on the SDK: https://github.com/stripe/stripe-android/blob/f440735d17c37477da0c73e58159f229803fc56a/financial-connections/src/androidTest/java/com/stripe/android/financialconnections/navigation/DestinationMappersTest.kt#L10

Reply:
I found I missed some enum cases when the API introduced new enum values and `when` ensures all cases are handled, but since you have test here it should be fine too

Reply:
if a new enum is introduced, the `DestinationMappersTest` would fail unless we specifically declare the pane in the `nonImplementedPanes` list